### PR TITLE
Automated run-by-run QA package (version-2) for STAR BES-II dataset QA.

### PR DESCRIPTION
**Pull request by EllipticFlow**
Automated run-by-run QA package (version-2) for STAR BES-II dataset QA.

### CODE REVIEW

File: StRoot/PWGTools/BadRunQA/qahist.root
Comment (plexoos) [inline StRoot/PWGTools/BadRunQA/qahist.root:1]: Can we please avoid adding binary files in the repository?

**Comment by marrbnl** [comment]
Just want to make sure I understand. This pull request is for the purpose of bookkeeping, and will be superseded by v3 from Tommy, right?

File: StRoot/PWGTools/BadRunQA/qahist.root
Comment (EllipticFlow) [reply StRoot/PWGTools/BadRunQA/qahist.root:1]: sure, let me then do the following, remove the binary files, uploaded to rcf, and add one more line in README so if people still want to test it, people can go and copy the binary file from there.

**Comment by EllipticFlow** [comment]
> 

yes, this pull request is for the purpose of bookkeeping. Will be superseded by v3 from Tommy.

File: StRoot/PWGTools/BadRunQA/qahist.root
Comment (EllipticFlow) [reply StRoot/PWGTools/BadRunQA/qahist.root:1]: ok, I did a new commit and push.

**Review by marrbnl — APPROVED** [review]

**Review by plexoos — APPROVED** [review]

**Pull request merged**